reference/gdscript_printf.rst

```diff
@@ -0,0 +1,194 @@
+    
+    print(actual_string)
+    # output: "We're waiting for Godot."
+
+Placeholders always start with a ``%``, but the next character or characters,
+the *format specifier*, determines how the given value is converted to a
+string.
+
+The ``%s`` seen in the example above is the simplest placeholder and works for
+most use cases: it converts the value by the same method by which an implicit
+String conversion or ``str()`` would convert it. Strings remain unchanged,
+Booleans turn into either ``"True"`` or ``"False"``, an integral or real number
+becomes a decimal, other types usually return their data in a human-readable
+string.
+
+There are other `format specifiers`_.
+
+
+Multiple placeholders
+---------------------
+
+Format strings may contain multiple placeholders. In such a case, the values
+are handed in the form of an array, one value per placeholder (unless using a
+format specifier with ``*``, see `dynamic padding`_)::
+
+    var format_string = "%s was reluctant to learn %s, but now he enjoys it."
+    var actual_string = format_string % ["Estragon", "GDScript"]
+    
+    print(actual_string)
+    # output: "Estragon was reluctant to learn GDScript, but now he enjoys it."
+
+Note the values are inserted in order. Remember all placeholders must be
+replaced at once, so there must be an appropriate number of values.
+
+
+Format specifiers
+-----------------
+
+There are format specifiers other than ``s`` that can be used in placeholders.
+They consist of one or more characters. Some of them work by themselves like
+``s``, some appear before other characters, some only work with certain
+values or characters.
+
+
+Placeholder types
+~~~~~~~~~~~~~~~~~
+
+One and only one of these must always appear as the last character in a format
+specifier. Apart from ``s``, these require certain types of parameters.
+
++-------+---------------------------------------------------------------------+
+| ``s`` | **Simple** conversion to String by the same method as implicit      |
+|       | String conversion.                                                  |
++-------+---------------------------------------------------------------------+
+| ``c`` | A single **Unicode character**. Expects an unsigned 8-bit integer   |
+|       | (0-255) for a code point or a single-character string.              |
++-------+---------------------------------------------------------------------+
+| ``d`` | A **decimal integral** number. Expects an integral or real number   |
+|       | (will be floored).                                                  |
++-------+---------------------------------------------------------------------+
+| ``o`` | An **octal integral** number. Expects an integral or real number    |
+|       | (will be floored).                                                  |
++-------+---------------------------------------------------------------------+
+| ``x`` | A **hexadecimal integral** number with **lower-case** letters.      |
```

Comment:
An "integral" is a noun, "integral" in "integral numer" is an adjective, meaning:

> being [...] one or more mathematical integers
> http://www.merriam-webster.com/dictionary/integral